Makefile

```diff
@@ -109,6 +110,14 @@ PLUGIN_PACKAGES += mattermost-plugin-nps-v1.1.0
 PLUGIN_PACKAGES += mattermost-plugin-welcomebot-v1.2.0
 PLUGIN_PACKAGES += mattermost-plugin-zoom-v1.5.0
 
+ifneq (Darwin,$(PLATFORM))
+	DOCKER_MAKEFILE := ./docker-compose.makefile.yml
+else ifeq (arm64, $(ARCH))
+    DOCKER_MAKEFILE := ./docker-compose.makefile.m1.yml
+else
+	DOCKER_MAKEFILE := ./docker-compose.makefile.yml
+endif
```

Comment:
I'm open for a better way to write this, it is the only way I found to make the tool happy and still get it to work

Reply:
💅 

```shell
DOCKER_MAKEFILE = ./docker-compose.makefile.yml

ifeq (arm64, $(ARCH))
    DOCKER_MAKEFILE = ./docker-compose.makefile.m1.yml	
endif
```

Reply:
there are other arm64 OSes, not sure if it makes sense to group those as the M1

Reply:
Ignore me i thought that was a PLATFORM check as well.